front/app/containers/Admin/reporting/components/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { IAppConfigurationData } from 'services/appConfiguration';
+
+export const shouldShowReportBuilderTab = (
+  appConfig: IAppConfigurationData
+) => {
+  const isReportBuilderAllowed =
+    appConfig.attributes.settings.report_builder?.allowed;
+
+  const isReportBuilderEnabled =
```

Comment:
I ended up reverting to this implementation in this file as these two are easier to reason about separately. Combining them makes the code hard to read and to reason about